test/test_images/sendevents/main.go

```diff
@@ -31,26 +31,24 @@ import (
 )
 
 var (
-	sink          string
-	eventID       string
-	eventType     string
-	eventSource   string
-	eventExtensionName     string
-	eventExtensionValue   string
-	eventData     string
-	eventEncoding string
-	periodStr     string
-	delayStr      string
-	maxMsgStr     string
+	sink                  string
+	eventID               string
+	eventType             string
+	eventSource           string
+	eventExtensionsMapStr string
```

Comment:
how does `eventExtensions` sounds like?
And then you can call the unmarshalled map `extensions`